.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,154 @@
+name: Automation - Prepare Release
+
+on:
+  workflow_dispatch:
+    # Determine the version number that will be assigned to the release. During the beta phase, we increment
+    # the minor version number and set the patch number to 0.
+    inputs:
+      candidate-stable:
+        required: true
+        description: Release candidate version (stable)
+      current-stable:
+        required: true
+        description: Current version (stable)
+      candidate-beta:
+        required: true
+        description: Release candidate version (beta)
+      current-beta:
+        required: true
+        description: Current version (beta)
+jobs:
+  # Releasing opentelemetry-collector
+  prepare-release:
+    runs-on: ubuntu-latest
+    steps:
+      # To keep track of the progress, it might be helpful to create a tracking issue similar to #6067. You are responsible
+      # for all of the steps under the "Performed by collector release manager" heading. Once the issue is created, you can
+      # create the individual ones by hovering them and clicking the "Convert to issue" button on the right hand side.
+      - name: Create issue for tracking release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CANDIDATE_BETA: ${{ inputs.candidate-beta }}
+          CANDIDATE_STABLE: ${{ inputs.candidate-stable }}
+          REPO: codeboten/opentelemetry-collector
+        run: |
+          EXISTING_ISSUE=$( gh issue list --search "Release v${CANDIDATE_BETA}/v${CANDIDATE_STABLE}" --json url --jq '.[].url' --repo ${REPO} )
+          if [ "${EXISTING_ISSUE}" != "" ]; then
+            echo "Issue already exists: ${EXISTING_ISSUE}"
+            exit 0
+          fi
+          gh issue create -a ${GITHUB_ACTOR} --repo ${REPO} --label release --title "Release v${CANDIDATE_BETA}/v${CANDIDATE_STABLE}" --body "Like #4522, but for v${CANDIDATE_BETA}/v${CANDIDATE_STABLE}
+          **Performed by collector release manager**
+
+          - [ ] Prepare stable core release v${CANDIDATE_STABLE}
+          - [ ] Prepare beta core release v${CANDIDATE_BETA}
+          - [ ] Tag and release stable core v${CANDIDATE_STABLE}
+          - [ ] Tag and release beta core v${CANDIDATE_BETA}
+          - [ ] Prepare contrib release v${CANDIDATE_BETA}
+          - [ ] Tag and release contrib v${CANDIDATE_BETA}
+          - [ ] Prepare otelcol-releases v${CANDIDATE_BETA}
+          - [ ] Release binaries and container images v${CANDIDATE_BETA}
+
+          **Performed by operator maintainers**
+
+          - [ ] Release the operator v${CANDIDATE_BETA}
+
+          **Performed by helm chart maintainers**
+
+          - [ ] Update the opentelemetry-collector helm chart to use v${CANDIDATE_BETA}"
+      # Make sure that there are no open issues with release:blocker label in Core. The release has to be delayed until they are resolved.
+      - name: Check blockers in core
```

Comment:
Should this happen even before creating the tracker issue?

Reply:
I think it should be fine either ways, since the issue will only get created once.